regexes/bots.yml

```diff
@@ -1005,6 +1035,11 @@
     name: 'QueryEye Inc.'
     url: 'http://queryeye.com'
 
+- regex: 'QuiteRSS'
+  name: 'QuiteRSS'
+  category: 'Feed Reader'
+  url: https://quiterss.org
```

Comment:
Are QuiteRSS and the other Feed Readers you added here online services that are crawling websites to present their feeds, or are they applications, that directly scrawl feeds on demand? If it's the latter one, we should add them as feed reader client instead

Reply:
You're right, I'll update it in the coming days

Reply:
@sgiehl I moved QuiteRSS as a feed reader app